Automattic/wp-calypso
# Sites Management Page: Filter site status using drop down instead of tabs

#### Proposed Changes

Part of @SaxonF's design changes proposed in https://github.com/Automattic/wp-calypso/pull/65871

* Removes site status tabs!
* Filter site status using a drop down.
* Big changes to the `useSitesTableFiltering` hook which no longer returns tab definitions
   * fuzzy search now performed _after_ status filtering so that the counts aren't affected by the query string (this was how Saxon's changes worked)
   * return counts for each status so it can be rendered in `<SitesDashboard>`
   * Updated status labels to match Saxon's design changes
* Update style of search box so it sites nicely beside the drop down
* Moved the "no sites" message so that they appear underneath the filtering controls. Otherwise there's no way to switch the filter to something else that might have sites.
* Add tests for `useSitesTableFiltering` hook and `<NoSitesMessage>` component, since that's where I've changed logic

![CleanShot 2022-08-03 at 22 08 24@2x](https://user-images.githubusercontent.com/1500769/182583283-e3e759c9-eb39-40f2-8f75-96fb1e1353ff.png)

#### Testing Instructions



* Check it out at `/sites-dashboard`
* Ensure the drop down control is always fully visible (I fixed issues where it was being cropped)
* Status counts shouldn't update as user types in search queries
* Ensure mobile layout is working

#### Pre-merge Checklist






Related to #65871
